agent/go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/orcaman/concurrent-map v0.0.0-20210106121528-16402b402231
 	github.com/ory/kratos-client-go v0.8.2-alpha.1
 	github.com/patrickmn/go-cache v2.1.0+incompatible
+	github.com/stretchr/testify v1.7.0
```

Comment:
is it needed for prod?
if this is a dev only dependency we should separate it

Reply:
This thread shed some light on how the go modules working:
https://github.com/golang/go/issues/26913